### PR TITLE
Add demo code

### PR DESCRIPTION
**Pull request by maryx**
Adds demo code that we use during the screencasts

@mjohnsullivan - you'll be demoing the files that starts with lesson_2_*

### CODE REVIEW

File: course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart [hunk 1]
```diff
@@ -1,46 +1,34 @@
...
-                textAlign: TextAlign.center,
-              ),
-            ),
-          ),
-        ),
-      ),
-    );
-  }
+class HelloRectangle extends StatelessWidget {
+ @override
```
Comment (yjbanov) [inline course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart]: nit: 2-space indent

File: course/code-snippets/lesson_1_09_demo.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        left: 32.0,
+        bottom: 32.0,
+        right: 32.0,
+      ),
+      child: container,
+    );
+  }
+}
+
+/// Example of a widget with the `children` property
```
Comment (yjbanov) [inline course/code-snippets/lesson_1_09_demo.dart]: nit: end sentence with a period.

File: course/code-snippets/lesson_1_20_demo.dart [hunk 3]
```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+void main() {
```
Comment (yjbanov) [inline course/code-snippets/lesson_1_20_demo.dart]: nit: indent using 2 spaces (applies throughout the file)

File: course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart [hunk 4]
```diff
@@ -1,46 +1,34 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
```
Comment (yjbanov) [inline course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart]: Unsure if we want to remove the license header.

**Comment by yjbanov** [comment]
LGTM with comments.

lgtm

File: course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart (hunk 4 above)
Comment (maryx) [reply course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart]: This was a mistake.

File: course/code-snippets/lesson_1_09_demo.dart (hunk 2 above)
Comment (maryx) [reply course/code-snippets/lesson_1_09_demo.dart]: Thanks!

File: course/code-snippets/lesson_1_20_demo.dart (hunk 3 above)
Comment (maryx) [reply course/code-snippets/lesson_1_20_demo.dart]: Ah oops

File: course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart (hunk 1 above)
Comment (maryx) [reply course/01_hello_rectangle/solution_01_hello_rectangle/lib/main.dart]: Done

**Pull request merged**